src/booleans.js

```diff
@@ -1,64 +1,92 @@
+
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b; 
 };
 
 function none(a, b) {
-  // your code here
+  if (a === false && b === false) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You can still use the shorthand as earlier for this:
```suggestion
  return !a && !b;
```